samples/boards/sensortile_box_pro/sensors-on-board/prj.conf

```diff
@@ -0,0 +1,14 @@
+CONFIG_LOG=y
```

Comment:
since you are enabling loggin you may as well use LOG_INF and get cached output and less chances of mixing up the messages? undecided, up to you, worth trying out, not sure it's work your fancy clear screen :-)

Reply:
Well, I'm not sure either... but I'll get a try. :D